pkg/virt-controller/watch/application.go

```diff
@@ -1064,6 +1070,12 @@ func (vca *VirtControllerApp) AddFlags() {
 
 	flag.IntVar(&vca.cloneControllerThreads, "clone-controller-threads", defaultControllerThreads,
 		"Number of goroutines to run for clone controller")
+
+	flag.StringSliceVar(&vca.additionalLauncherAnnotationsSync, "additional-launcher-annotations-sync", []string{},
+		"Comma separated list of annotation keys which if present on the VM template and so VMI, will be sync to the virt-launcher pod. Note, it is unidirectional from VM.spec.template.metadata -> VMI -> virt-launcher pod")
```

Comment:
nit: `VM.spec.template.metadata -> VMI and VMI -> virt-launcher pod separately`

I think users can use VMI -> virt-launcher label sync separately from VM -> VMI -> virt-launcher.

Reply:
Not really: this only applies when the VMI is not associated with a VM.
When a VMI is tied to a VM, any managed label added directly to the VMI but missing from the VM’s `spec.template.metadata` will be stripped from the VMI.
This behavior is also covered by unit test `should remove selected custom labels from VMI if missing in VM.spec.template.metadata.labels` in `vm_test.go`

Reply:
> Not really: this only applies when the VMI is not associated with a VM.

yes this is what I am talking about. The doc here doesnt specific it works for VMI not associated with a VM

Reply:
done